setup/install/providers/appengine.md

```diff
@@ -120,18 +120,10 @@ fetch your tar archive, untar it, and then deploy the code to App Engine.
 ### Deploying from Google Container Registry URL
 
 Spinnaker supports deploying Docker containers on the App Engine Flex runtime from images built and stored
-in Google Container Registry from just a gcr.io URL.  This feature is currently flagged because it is still quite new.
+in Google Container Registry from just a gcr.io URL.
 
-In order to enable this feature, set the flag with Halyard like so:
-
-```bash
-hal config features edit --appengine-container-image-url-deployments true
-```
-
-After doing this you'll find an option in the Create Server Group modal in Deck to use a Container Image as a
-deployment's Source Type.
-
-Selecting the Container Image option reveals a textbox that can then be used to specify the gcr.io URL.  Alternatively
+You'll find an option in the Create Server Group modal in Deck to use a Container Image as a
+deployment's Source Type. Selecting the Container Image option reveals a textbox that can then be used to specify the gcr.io URL.  Alternatively
```

Comment:
If Container Image and Source Type are UI elements, make them bold face, but if they're not, change them to lower case.

Reply:
👍 Done. Thanks for the fast review, @dorbin!